infoTree/out/url114.txt

```diff
@@ -0,0 +1,20 @@
+{
+	"title": "Oil-Based Paint",
+	"heading": "Hazardous ? no charge",
+	"para": {
+		"text": "Bring oil-based paint to a ",
+		"link": {"href": "../../../../../MyServices/Garbage/SpecialorHazardousItems/HouseholdHazardousWaste/WheretoDisposeofHazardousWaste/index.htm", "text": "household hazardous waste facility"},
+		"text": ". No fee."
+
+	},
+	"heading": "In garbage",
+	"para": "Empty oil-based paint cans can go in the garbage:",
+	{
+		"bullet": "Remove the lid and let fumes evaporate outside, away from children and animals.",
+		"bullet": "Separate the lid from the dried paint can and put both in your garbage."
```

Comment:
So all of the bullets were <ul> tags in html document, but since some of them do seem like they're ordered, we can always go in and manually edit the JSON (or JSX at this point) to render an ordered list instead